browser/src/VariantPage/VariantInSilicoPredictors.tsx

```diff
@@ -93,7 +118,9 @@ const VariantInSilicoPredictors = ({ variant, datasetId }: Props) => {
               const predictor = predictors[id]
 
               let color = null
-              const parsedValue = parseFloat(value)
+              const parsedValue = predictor.absoluteValue
```

Comment:
i think using the absolute value here is probably reasonable here, as long as we know the thresholds are always going to be the same on either side of 0.

i.e., if the need to specify:

`pangolin_largest_ds: { label: 'Pangolin', warningThreshold: 0.2, dangerThreshold: 0.5 },`

and

`pangolin_largest_ds: { label: 'Pangolin', warningThreshold: -0.3, dangerThreshold: 0.6 },`

where the thresholds are different numbers for positive and negative, we might need something different here.

But, disregard if those will always match 👍

Reply:
Yeah good eye, this is the first one that even needed negative value thresholds. For this instance negative denotes a different class of prediction (splice gained vs splice loss) that shares the same thresholds. Without knowing I'd hope its a fair guess that other predictors may follow a similar pattern if they allow negative scores?

We can revisit later down the line if needed.